Multisig/Logic/Ethereum/GnosisSafe.swift

```diff
@@ -28,6 +28,10 @@ class GnosisSafe {
 
     var fallbackHandlers:[(fallbackHandler: Address, label: String)] = [("0xd5D82B6aDDc9027B22dCA772Aa68D5d74cdBdF44", "DefaultFallbackHandler")]
 
+    func hasFallbackHandler(safe: Safe) -> Bool {
+        !(safe.fallbackHandler?.isZero ?? true)
```

Comment:
This is not easy to understand... Can you please simplify a bit to remove the negation of the ??

`safe.fallbackHandler.map { !$0.isZero } ?? false`

or

`safe.fallbackHandler?.isZero == false`

or 

`safe.fallbackHandler != nil  && safe.fallbackHandler != .zero`